pkg/serializer/jsonstream/payload_builder_nozlib.go

```diff
@@ -14,6 +14,17 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/serializer/marshaler"
 )
 
+// OnErrItemTooBigPolicy defines the behavior when OnErrItemTooBig occurs.
+type OnErrItemTooBigPolicy int
+
+const (
+	// ContinueOnErrItemTooBig when founding an ErrItemTooBig, skips the error and continue
+	ContinueOnErrItemTooBig OnErrItemTooBigPolicy = iota
```

Comment:
note to include any change made in the regular payload_builder.go

Reply:
What do you mean?

Reply:
I mean changes need to be mirrored between `payload_builder.go` and `payload_builder_nozlib.go`